tests/test_prior_d2.py

```diff
@@ -0,0 +1,83 @@
+from torchmdnet.priors import D2
+import torch as pt
+from pytest import mark
+
+
+def compute_with_psi4(z, pos):
```

Comment:
This doesn't seem to be used anywhere.  Is it just to document how you computed the values below?  Maybe add a comment saying that?

Reply:
Yes, add a comment.